crates/sargon-uniffi/src/signing/authentication_signing_input.rs

```diff
@@ -0,0 +1,13 @@
+use crate::prelude::*;
+use sargon::AuthenticationSigningInput as InternalAuthenticationSigningInput;
+
+#[derive(Clone, PartialEq, InternalConversion, uniffi::Record)]
```

Comment:
Can't remember, does InternalConversion also generate sample
Values? Else maybe we wanna create and UniFFI export sample
Values!?

Reply:
Did that, will push soon.

Reply:
And that goes for all UniFFI exported types :)